mappings/i2b2/condition-mapping.json

```diff
@@ -50,19 +50,23 @@
         "resourceType" : "Condition",
         "code" : {
           "coding" : [ {
-            "code" : "{{%icd10Code.target_code}}",
-            "system" : "{{%icd10Code.target_system}}",
+            "code" : "{{%icd10Code.first().target_code}}",
```

Comment:
I select the first matching code, but since we have multiple mappings, how should we handle this? Should we create a separate Procedure resource for each code, or include all of them within a single Procedure resource?

Reply:
IMO the best approach is to generate a single Procedure resource and include all the mapped codes into that single resource. Because, the intent of the source health data is to represent a single procedure entity, hence we need to represent the same in our mapped FHIR resources.